iris-mpc-cpu/src/execution/hawk_main.rs

```diff
@@ -843,27 +845,32 @@ impl HawkResult {
         let match_ids = self.select_indices(Filter {
             eyes: Both,
             orient: Only(Normal),
+            intra_batch: false,
         });
 
         let (partial_match_ids_left, partial_match_counters_left) = self.select(Filter {
             eyes: Only(Left),
             orient: Only(Normal),
+            intra_batch: true,
```

Comment:
AFAIK, the batch_matches are not put into the partial match ids/counters:
https://github.com/worldcoin/iris-mpc/blob/e947d73284643df2c9c32d5e6717103326771697/iris-mpc-gpu/src/dot/kernel.cu#L258-L294 has no reference to these passed in compared to the above `merge*` functions.

Reply:
This of course also follows for right and mirrored variants of partial matches.

Reply:
Thanks for the info, done!